mlrun/utils/helpers.py

```diff
@@ -470,11 +499,35 @@ def get_parsed_docker_registry() -> Tuple[Optional[str], Optional[str]]:
         )
 
 
+def is_store_uri(url):
+    """detect if the uri starts with the store schema prefix"""
+    return url.startswith(DB_SCHEMA + "://")
+
+
+def parse_store_uri(url):
```

Comment:
Even though it's a pretty simple function, it's widely used, add tests for it - see `test_enrich_image` for an example

Reply:
done